data/brands/shop/supermarket.json

```diff
@@ -9650,6 +9653,62 @@
         "name:ja": "関西スーパー",
         "shop": "supermarket"
       }
+    },
+    {
+      "displayName": "イオンスーパーセンター",
+      "locationSet": {"include": ["jp"]},
+      "tags": {
+        "brand": "AEON SUPERCENTER",
```

Comment:
```suggestion
        "brand": "イオンスーパーセンター",
```
I‘ll change it to katakana because its website all use kana, not English

![图片](https://user-images.githubusercontent.com/42690037/191607234-b5c40d25-8db6-4e2a-b333-c6fb5fae4ee6.png)

Like your show cases in https://github.com/osmlab/name-suggestion-index/pull/7055/files#r975374264 and https://github.com/osmlab/name-suggestion-index/pull/7055/files#r976153435, those brands are using more latin character in everywhere, but this AEON SUPERCENTER looks like more localized.

And for name tag, I search for wikipedia, [イオンスーパーセンター](https://ja.wikipedia.org/wiki/%E3%82%A4%E3%82%AA%E3%83%B3%E3%82%B9%E3%83%BC%E3%83%91%E3%83%BC%E3%82%BB%E3%83%B3%E3%82%BF%E3%83%BC) and [イオン北海道](https://ja.wikipedia.org/wiki/%E3%82%A4%E3%82%AA%E3%83%B3%E5%8C%97%E6%B5%B7%E9%81%93)

Reply:
I see. Thank you very much.

Regarding the AEON SUPERCENTER brand tag, I am again referring to the value of AEON in the JA:Naming sample.
Also, looking at the [official site policy](https://www.aeonsupercenter.co.jp/contact_s/site-policy/) and the [national trademark search platform](https://www.j-platpat.inpit.go.jp/t0201), it appears that the actual trademark registration is alphabetical.

I thought it would be better to use alphabetical notation because the Wiki says that the value used in the brand tag is usually a trademark, and in Japan, data users can use either notation by referring to both the brand and the brand:ja tags.